Client/Application/AppDelegate.swift

```diff
@@ -179,6 +201,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // We sync in the foreground only, to avoid the possibility of runaway resource usage.
     // Eventually we'll sync in response to notifications.
     func applicationDidBecomeActive(application: UIApplication) {
+        // handle quick actions is available
+        if #available(iOS 9, *) {
+            guard let shortcut = quickActions.launchedShortcutItem else { return }
+            quickActions.handleShortCutItem(shortcut, completionBlock: handleShortCutItemType)
+            quickActions.launchedShortcutItem = nil
```

Comment:
Do we want to add a return statement at the end of this block in case we add more to this method and don't want to passthrough?